dspace-server-webapp/src/main/java/org/dspace/app/rest/health/SEOHealthIndicator.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.health;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Implementation of {@link org.springframework.boot.actuate.health.HealthIndicator} that verifies if the SEO of the
+ * DSpace instance is configured correctly.
+ *
+ * This is only relevant in a production environment, where the DSpace instance is exposed to the public.
+ */
+public class SEOHealthIndicator extends AbstractHealthIndicator {
+
+    ConfigurationService configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+
+    private final RestTemplate restTemplate = new RestTemplate();
+
+    @Override
+    protected void doHealthCheck(Health.Builder builder) {
+        String baseUrl = configurationService.getProperty("dspace.ui.url");
+
+        boolean sitemapOk = checkUrl(baseUrl + "/sitemap_index.xml") || checkUrl(baseUrl + "/sitemap_index.html");
+        RobotsTxtStatus robotsTxtStatus = checkRobotsTxt(baseUrl + "/robots.txt");
+        boolean ssrOk = checkSSR(baseUrl);
+
+        if (sitemapOk && robotsTxtStatus == RobotsTxtStatus.VALID && ssrOk) {
+            builder.up()
+                   .withDetail("sitemap", "OK")
+                   .withDetail("robots.txt", "OK")
+                   .withDetail("ssr", "OK");
+        } else {
+            builder.down();
+            builder.withDetail("sitemap", sitemapOk ? "OK" : "Sitemaps are missing or inaccessible. Please see the " +
+                    "DSpace Documentation on Search Engine Optimization for how to enable Sitemaps.");
+
+            if (robotsTxtStatus == RobotsTxtStatus.MISSING) {
```

Comment:
Tiny thing, this `if` , `else if` needs a `else` that says "OK".  Currently, if something else fails, then Robots.txt is missing from the output.

For instance, here's what it looks like if Sitemaps are not setup:
![no-sitemaps](https://github.com/user-attachments/assets/e4ff4aeb-ac46-48cd-b9c9-53f25f0bc6c1)

Notice that "Robots.txt" is missing.  The reason is that it is neither missing nor invalid (and this if/else is missing the "OK" result)

Reply:
Thanks for catching this, I indeed missed the success status here for robots.txt